packages/@uppy/dashboard/src/Dashboard.tsx

```diff
@@ -170,7 +170,7 @@ interface DashboardMiscOptions<M extends Meta, B extends Body>
   thumbnailHeight?: number
   thumbnailType?: string
   thumbnailWidth?: number
-  trigger?: string | Element
+  trigger?: string | Element | null
```

Comment:
for people who have a strict tsconfig and have set `exactOptionalPropertyTypes: true`, it's nice to also allow undefined (but this also applies to all other options, so maybe a separate pr when i think about it..)
```suggestion
  trigger?: string | Element | null | undefined
```

Reply:
imo that's what `?` is for, we don't need to add `undefined` everywhere. Too redundant.

Reply:
that is incorrect when `exactOptionalPropertyTypes: true`:

- `undefined` means that the property exists on the object but has the value `undefined`, e.g. `{ prop: undefined }` satisfies the type `{ prop: string | undefined }`
- `?` means that the property might not exist at all, e.g.: `{}` satisfies the type `{ prop?: string }` 

And we cannot control whether or not consumers of uppy are using `exactOptionalPropertyTypes: true` in their tsconfig